package.json

```diff
@@ -76,7 +76,7 @@
     "@bbc/psammead-locales": "^4.1.1",
     "@bbc/psammead-media-indicator": "^2.6.19",
     "@bbc/psammead-media-player": "^2.4.1",
-    "@bbc/psammead-navigation": "^3.0.5",
+    "@bbc/psammead-navigation": "^6.0.0-alpha.22",
```

Comment:
We need to bring this out of alpha - it has now passed a11y review and there are tickets to fix any non P1 bugs after we go live.